public/javascripts/Admin/src/Admin.js

```diff
@@ -1150,13 +1150,93 @@ function Admin(_, $) {
         } )
     }
 
+    function convertDistance(distance) {
```

Comment:
Instead of doing this yourself here, we have some utility functions written to do this, in `UtilitiesMath.js`! You can add an import of that file for the admin page using `Gruntfile.js`!

Reply:
Sounds good! I didn't see any methods that took the current measurement system & returned some transformation on the distance, so I kept this method in and called the utils method in the body.

Reply:
Makes sense!